src/questions/venus.yaml

```diff
@@ -0,0 +1,229 @@
+- uuid: 9e285187-2ae0-42dc-8b91-e36926634f47
+  type: SingleChoice
+  content: Which of these features are available in Venus?
+  hint: NoHint
+  explanation: Venus has the features of a money market protocol and stablecoin minting protocol together.
+  answerKeys:
+    - D
+  difficultyLevel: "Low"
+  choices:
+    - content: Money market features
+      key: A
+      
+    - content: Stablecoin minting feature
+      key: B
+      
+    - content: Permissionless listing
+      key: C
+      
+    - content: Both A and B
+      key: D
+      
+- uuid: db7b0642-1254-40c1-955a-4f50af3fd615
+  type: SingleChoice
+  content: What are the usecases of vTokens?
+  hint: NoHint
+  explanation: vTokens are used to accrue interest for the user supplied collateral. They can be used as collateral to borrow.
+  answerKeys:
+    - A
+  difficultyLevel: "Medium"
+  choices:
+    - content: To accrue interest on assets supplied and be used as collateral
+      key: A
+      
+    - content: To accrue debt and be used as collateral
+      key: B
+      
+    - content: To accrue debt only
+      key: C
+      
+    - content: To accrue debt and interest for borrowed and supplied assets respectively
+      key: D
+      
+- uuid: 6d862849-fcb2-4d76-b96c-170f556d5c72
+  type: SingleChoice
+  content: The Mint function in Venus is used to?
+  hint: NoHint
+  explanation: The mint function in venus is used to supply assets to the protocol and mint an equivalent amount of vTokens for the underlying asset.
+  answerKeys:
+    - C
+  difficultyLevel: "Medium"
+  choices:
+    - content: To mint their stable coin, VAI
```

Comment:
Is A a wrong option

Reply:
Mint (https://docs.venus.io/docs/vtokens#mint)
The mint function transfers an asset into the protocol, which begins accumulating interest based on the current 
Supply Rate for The user receives a quantity of vTokens equal to the underlying tokens tokens supplied, divided by the current [Exchange Rate](https://docs.venus.io/docs/vtokens#vtokens-exchange-rate).

Reply:
VAI is minted used mintVAI function